Gemfile

```diff
@@ -6,13 +6,14 @@ group :development, :test do
   gem 'puppetlabs_spec_helper',  :require => false
   gem 'puppet-lint',             :require => false
   gem 'simplecov',               :require => false
-  gem 'nokogiri',                '<= 1.5.10'
+  gem 'nokogiri',                :require => false
```

Comment:
yeah, we have to do that. cause nokogiri dropped support for ruby 1.8.7 at version 1.6.0, which is the next version after 1.5.10